skbio/workflow.py

```diff
@@ -220,6 +220,8 @@ def __init__(self):
     def __call__(self, msg):
         self.msg = msg
         return self
+
+
```

Comment:
Seems to be an update on pyflake. A fresh install will fail without these spaces